Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -270,6 +270,28 @@ LayoutUnit GridTrackSizingAlgorithm::initialGrowthLimit(const GridTrackSize& tra
     return infinity;
 }
 
+void GridTrackSizingAlgorithm::sizeTrackToFitSingleSpanMasonryGroup(const GridSpan& span, struct MasonryIndefiniteItems &masonryIndefiniteItems, GridTrack& track)
```

Comment:
Updating to `subgridMasonryIndefiniteItems` to give a better description.

We are trying to get all the children of the gridItem, so it might be a bit confusing, since we already have `masonryIndefiniteItems` above.